dataland-frontend/tests/e2e/utils/FakeFixtureUtils.ts

```diff
@@ -1,11 +1,11 @@
-import { faker } from "@faker-js/faker";
-
 /**
  * Returns either the input string or undefined with a 50/50 chance
  *
  * @param input the string to be returned
+ * @param undefinedPercentage the probability that this function returns undefined. Defaults to 50%
  * @returns either the input or undefined
  */
-export function randomStringOrUndefined(input: string): string | undefined {
-  return faker.helpers.arrayElement([undefined, input]);
+export function randomStringOrUndefined(input: string, undefinedPercentage = 0.5): string | undefined {
+  if (Math.random() < undefinedPercentage) return undefined;
```

Comment:
We have a very compact way of doing this in DataPointFixtures.ts:

return Math.random() > undefinedRatio ? value : undefined;

We could also think about consolidating these functionalities in the future, but not in this story.

Reply:
Fixed & Created a refactoring backlog item